docs/pages/error/runtime-in-config/+Page.mdx

```diff
@@ -3,49 +3,54 @@ import { Link, Construction } from '@brillout/docpress'
 If you get an error like this:
 
 ```
-[Wrong Usage] title defined by /pages/+config.js must be defined using a separate file +title.js
+[Wrong Usage] title defined by /pages/+config.ts must be defined using a separate file +title.ts
```

Comment:
Let's only use `.ts` if we will be able to apply to upcoming `JS<->TS` toggle. I went ahead and made the change.